build/build_app_engine_package.py

```diff
@@ -95,14 +95,28 @@ def main(src_path, dest_path):
 
   build_version_info_file(os.path.join(dest_path, 'version_info.json'))
 
-if __name__ == '__main__':
+
+def CopyTests(src_path, dest_path):
+  for dirpath, _, files in os.walk(src_path):
+    if dirpath.endswith('app_engine'):
+      tests = [name for name in files if 'test' in name]
+      for test in tests:
+        shutil.copy(os.path.join(dirpath, test), dest_path)
+
+
+def main():
   parser = optparse.OptionParser(USAGE)
+  parser.add_option("-t", "--include-tests", action="store_true",
+                    help='Also copy python tests to the out dir.')
   options, args = parser.parse_args()
   if len(args) != 2:
-    print 'Error: Exactly 2 arguments required.'
-    parser.print_help()
-    sys.exit(1)
+    parser.error('Error: Exactly 2 arguments required.')
 
-  SRC_PATH = args[0]
-  DEST_PATH = args[1]
-  main(SRC_PATH, DEST_PATH)
+  src_path, dest_path = args[0:2]
+  CopyApprtcSource(src_path, dest_path)
+  if options.include_tests:
+    CopyTests(src_path, dest_path)
```

Comment:
The tests are copied but never cleaned up, so if someone run the python tests once, the test files will remain in the out folder, and it's error prone to push all of them to GAE.